security/encrypt-cluster-pod-traffic.md

```diff
@@ -87,27 +89,28 @@ To install WireGuard for OpenShift v4.3:
    EOF
    ```
 
-   3. Configure files.
+   3. Download and configure the tools needed for kmods.
    ```bash
    FAKEROOT=$(mktemp -d)
    git clone https://github.com/kmods-via-containers/kmods-via-containers
    cd kmods-via-containers
    make install DESTDIR=${FAKEROOT}/usr/local CONFDIR=${FAKEROOT}/etc/
    cd ..
-   git clone https://github.com/realgaurav/kvc-wireguard-kmod
+   git clone https://github.com/projectcalico/kvc-wireguard-kmod
    cd kvc-wireguard-kmod
    make install DESTDIR=${FAKEROOT}/usr/local CONFDIR=${FAKEROOT}/etc/
    cd ..
    ```
 
-   4. Configure RPMs for kernel-core, kernel-devel and kernel-modules for the host kernel (can be found by running uname -r on the host). Update `$FAKEROOT/etc/kvc/wireguard-kmod.conf` for the RPM location.
+   4. You must then set the value for the `KERNEL_CORE_RPM`, `KERNEL_DEVEL_RPM` and `KERNEL_MODULES_RPM` packages in the conf file `$FAKEROOT/etc/kvc/wireguard-kmod.conf`. These will be URLs to the RPMs. You can determine the host kernel version to use in the URL by running `uname -r` on a host in your cluster. You can find links to  official packages in your Red Hat subscription at https://access.redhat.com/downloads/content/package-browser 
```

Comment:
suggest making https://access.redhat.com a live link that opens in a new window.

Also, what happens if the user fails to set the `KERNEL` values? Would be great if the process fails in an easily debugged way.

Reply:
It will fail on the apply step of the machine config to the cluster. I think we can modify the shell script to fail faster and earlier in the repo these instructions references, to give a better user experience. These docs would still be okay and not require a change.